lib/workers/global/limits.ts

```diff
@@ -27,7 +28,7 @@ export function incLimitedValue(key: Limit, incBy = 1): void {
   });
 }
 
-export function isLimitReached(key: Limit): boolean {
+function handleCommitsLimit(key: Limit): boolean {
```

Comment:
It makes no sense to rename this `handleCommitsLimit()` but still require `key` (which can only be `Commits`). Either keep it the same name but with one possible key (in case we have new keys in future), or do the rename like this but drop the `key` param.